src/client/gfs_client.cc

```diff
@@ -0,0 +1,40 @@
+#include "gfs_client.h"
+
+using namespace gfs;
+using namespace client;
+
+namespace gfs {
+namespace client {
+
+// This function should initialize objects such as the ClientStubs for between the
+// client and the master, and client and the chunk server. The param list needs to be
+// extended to include chunk server's information as well. 
+void configClient(const char* masterUrl) {
```

Comment:
Let's remove it  here for now, and hard code as necessary, and allocate this to configuration manager I will introduce later.

Reply:
Resolved.